framework/cluster/src/main/java/com/cloud/cluster/ClusterFenceManagerImpl.java

```diff
@@ -22,6 +22,7 @@
 import javax.inject.Inject;
 import javax.naming.ConfigurationException;
 
+import org.apache.cloudstack.management.ManagementServerHost;
```

Comment:
do you need this import here?

Reply:
Because of the interface move to `cloud-api`module.

Reply:
ok, got it.
This `ManagementServerHost` was already used as a return type of a method, that is why I did not see any other reference in the code, and only an import needed to be added.

Thanks!